terraspace/app/stacks/dns/outputs.tf

```diff
@@ -1,5 +1,4 @@
-# TODO: Replace with new domain to start advertising new address through publishing subsys. Will do after confirming with #storetheindex
 output "bitswap_loadbalancer_domain" {
-  value       = aws_route53_record.peer_bitswap_load_balancer.name
+  value       = "${cloudflare_record.bitswap_peer.name}.${data.cloudflare_zone.dns.name}"
```

Comment:
Cant we get the full name from the `cloudflare_record` resource?  I'd prefer it, if we can avoid having to concat the two resources.

Reply:
Done